sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md

```diff
@@ -149,6 +149,9 @@
 ### Major changes
 - SecretClient.vaultUrl has been added with the original value pass to SecretClient.
 
+## 4.0.0-preview.3 (2019-09-10)
```

Comment:
I don't know why `4.0.0-preview.4` has the date "2019-09-08" here, it looks different on my local version of `master` ("2019-10-08").
![image](https://user-images.githubusercontent.com/5799331/116500288-d6992c00-a873-11eb-9a7c-e33e69c6baa4.png)

Reply:
That's weird. Do you have the latest master.

Reply:
Do you have latest master. its is 2019-09-08` on github.